benchmark_runner/workloads/linstress_vm.py

```
@@ -0,0 +1,172 @@
+            for elapsed in range(0, self._timeout, OC.DELAY):
+                check = self._virtctl.virtctl_ssh(vm_name=vm_name, command=check_cmd,
+                                                  namespace=self.__namespace, key_path=self.__ssh_key_path,
+                                                  username=self.__username, timeout=60)
+                if check and 'found' in check:
+                    break
+                if elapsed > 0 and elapsed % 300 == 0:
+                    logger.info(f'Waiting for stress_report.json on VM {vm_name}... ({elapsed}s)')
+                time.sleep(OC.DELAY)
+
+            if not self._virtctl._scp_file(vm_name=vm_name,
+                                           remote_path='/tmp/stress_report.json',
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/stress_report.json"

(S108)